modules/develop/partials/run-transforms.adoc

```diff
@@ -333,10 +440,11 @@ See xref:reference:public-metrics-reference.adoc[]
 - Only a single output topic is supported.
 - Transforms have at-least-once delivery.
 - When clients use the Kafka Transactions API on partitions of an input topic, transforms process only committed records.
-- Because data transforms are powered by Wasm, transform functions can be authored in any language. However, a data transforms SDK currently is only available in xref:reference:data-transform-api.adoc[Golang].
+- Because data transforms are powered by Wasm, transform functions can be authored in any language. However, a data transforms SDK currently is only available in xref:reference:data-transform-api.adoc[Golang] and xref:reference:data-transform-rust-sdk.adoc[Rust].
 
 == Suggested reading
 
 - xref:./how-transforms-work.adoc[]
 - xref:reference:data-transform-api.adoc[]
+- xref:reference:data-transform-rust-sdk.adoc[]
```

Comment:
These read very different:

<img width="279" alt="Screenshot 2024-01-30 at 8 40 12 AM" src="https://github.com/redpanda-data/docs/assets/8669100/64350c12-67b1-441f-bcf1-fb2a45a87bd6">

Should we unify the names so it is clear they are the same thing for different languages?